src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -337,42 +139,39 @@ std::variant<Variable, ParsedQuery::Alias> SparqlQleverVisitor::visitTypesafe(
 }
 
 // ____________________________________________________________________________________
-ParsedQuery::Alias SparqlQleverVisitor::visitTypesafe(
-    SparqlAutomaticParser::AliasContext* ctx) {
+ParsedQuery::Alias Visitor::visitTypesafe(Parser::AliasContext* ctx) {
   // A SPARQL alias has only one child, namely the contents within
   // parentheses.
   return visitTypesafe(ctx->aliasWithoutBrackets());
 }
 
 // ____________________________________________________________________________________
-ParsedQuery::Alias SparqlQleverVisitor::visitTypesafe(
-    SparqlAutomaticParser::AliasWithoutBracketsContext* ctx) {
-  auto wrapper = makeExpressionPimpl(visit(ctx->expression()));
+ParsedQuery::Alias Visitor::visitTypesafe(
+    Parser::AliasWithoutBracketsContext* ctx) {
+  auto wrapper = makeExpressionPimpl(visitTypesafe(ctx->expression()));
   return ParsedQuery::Alias{std::move(wrapper), ctx->var()->getText()};
```

Comment:
There are quite a lot of redundant return types here, above and below, maybe use the chance to get rid of them:
`return {std::move(wrapper), ctx->var()->getText()};`